esp-box/esp-box_idf4.c

```diff
@@ -0,0 +1,67 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: CC0-1.0
```

Comment:
We are using same license in all BSP files.

Reply:
@espzav In that case, we should change it to Apache 2.0 license, since you have an apache license in esp-box component: https://github.com/espressif/esp-bsp/blob/c6ae6ebf02bbbadaf583503cea72a7912c2b3bb5/esp-box/LICENSE

Reply:
@igrr Thanks for clarify. I see, that we have it same in all BSPs. We should consider it. I will create ticket and I will make it. But problem was, I don't understand licesnes too much. I didn't know, which header should be there. 
Thank you for help.

Reply:
Fixed in #195